app/packages/core/src/plugins/SchemaIO/components/NativeModelEvaluationView/index.tsx

```diff
@@ -82,6 +94,43 @@ export default function NativeModelEvaluationView(props) {
     [triggerEvent, rename_evaluation, evaluations, onChange]
   );
 
+  const time = new Date().getTime();
+
+  const onDelete = useCallback(
+    (eval_id: string, eval_key: string) => {
+      triggerEvent(
+        delete_evaluation,
+        { eval_id, eval_key },
+        false,
+        (results) => {
+          if (results?.error === null) {
+            // update the current page display
+            const updatedEvaluations = evaluations.filter(
+              (evaluation) => evaluation.id !== eval_id
+            );
+            onChange("evaluations", updatedEvaluations);
+            onChange(
+              "view.key",
+              `${eval_id}_${updatedEvaluations.length}_${time}`
+            );
+            if (page === "evaluation") {
+              // should return to overview page
+              onChange("view", { page: "overview" });
+              onChange("view.key", eval_id + "_deleted");
```

Comment:
Does setting to `null`? If not, this may be limitation of framework we may want to address in near future